lib/scarlet/plugin.rb

```diff
@@ -34,6 +34,8 @@ def initialize event, *objs
 
     def exec(&block)
       catch(:abort) { instance_exec(@event, &block) }
+    rescue SystemExit => sysexit
```

Comment:
Just rescue `StandardError`, otherwise we make Scarlet freeze up on general signal handling... `SignalException` will silently fail